tests/vxlan/vnet_utils.py

```diff
@@ -0,0 +1,122 @@
+import json
+import logging
+import vnet_config as vc
+import yaml
+
+from jinja2 import Template
+from os import path
+from time import sleep
+
+logger = logging.getLogger(__name__)
+
+def gen_vnet_config(mg_facts, num_vnet, num_routes, num_endpoints):
+    """
+    @summary: Generates and stores the VNET configuration
+    @param mg_facts: Minigraph facts
+    @param num_vnet: Number of VNETs
+    @param num_routes: Number of routes
+    @param num_endpoints: Number of endpoints
```

Comment:
Please check https://github.com/Azure/sonic-mgmt/blob/master/tests/styleguide.md for the comment format.